src/test/java/tests/BasicTest.java

```diff
@@ -1,3 +1,5 @@
+package tests;
```

Comment:
I don't think this *really* matters, but there's certainly no reason to add it in this PR.